packages/react-composites/src/composites/CallComposite/styles/CallControls.styles.ts

```diff
@@ -1,19 +1,30 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { DefaultPalette as palette, IButtonStyles, IContextualMenuItemStyles, Theme } from '@fluentui/react';
+import { DefaultPalette as palette, IButtonStyles, IContextualMenuItemStyles, IStyle, Theme } from '@fluentui/react';
 import { OptionsButtonStyles, ParticipantsButtonStyles } from '@internal/react-components';
 
 const MINIMUM_TOUCH_TARGET_HEIGHT_REM = 3;
 
+/** @private */
+export const controlBarContainerStyles: IStyle = {
+  paddingTop: '0.25rem',
+  paddingBottom: '0.25rem',
+
+  // @TODO: this should be exposed through a custom CallComposite Theme API that extends the fluent theme with semantic values
+  boxShadow: `
+    0px 6.400000095367432px 14.399999618530273px 0px #00000021;
+    0px 1.2000000476837158px 3.5999999046325684px 0px #0000001A;
+  `
+};
+
 /**
  * @private
  */
 export const groupCallLeaveButtonStyle = {
   root: {
     border: '0.125rem',
     borderRadius: 2,
-    marginRight: '.75rem',
```

Comment:
anyone know why this was here? It caused our buttons to be slightly off-center

Reply:
Ah thanks! I will test it out then add this back as padding to the parent container (both left and right sides) to ensure buttons are centered

Reply:
Tested this and has correct behavior in composites and in the storybook